server/apps/highlights/service.py

```diff
@@ -72,17 +78,27 @@ def create(self, docs, **kwargs):
             highlights = item.get('highlights', [])
             if not highlights:
                 highlights = []
+
             if doc['highlights'] not in highlights:
                 highlights.append(doc['highlights'])
+                highlight_on = True  # highlight toggled on
             else:
                 highlights = [h for h in highlights if h != doc['highlights']]
+                highlight_on = False  # highlight toggled off
+
             updates = {
                 'highlights': highlights,
                 '_updated': item['_updated'],
                 '_etag': item['_etag']
             }
             service.update(item['_id'], updates, item)
-        push_notification('item:mark', marked=1)
+
+            push_notification(
+                'item:mark',
```

Comment:
I agree. I kept the existing name, though `"mark"` is indeed kind of too generic a label.